codespell_lib/data/dictionary.txt

```diff
@@ -32945,6 +32951,8 @@ legth->length
 legths->lengths
 leibnitz->leibniz
 leightweight->lightweight
+lement->element, clement, lament,
```

Comment:
Any typos of `cement` and `dement` out there?

Reply:
I couldn't find these typos.